include/third_party/ffts/ffts_attributes.h

```diff
@@ -0,0 +1,111 @@
+/*
+
+ This file is part of FFTS -- The Fastest Fourier Transform in the South
```

Comment:
@kiritigowda : Please verify including this as third party is OK from osrb requirements